### PR TITLE
Feat/reschedule appointment

### PR DESCRIPTION
**Pull request by LoisChen68**

### CODE REVIEW

**Comment by LoisChen68** [comment]
[(3)【我的主頁】學員前台我的主頁新增「更換時段」功能[2]](https://trello.com/c/nhpDgrOE/4626-3%E3%80%90%E6%88%91%E7%9A%84%E4%B8%BB%E9%A0%81%E3%80%91%E5%AD%B8%E5%93%A1%E5%89%8D%E5%8F%B0%E6%88%91%E7%9A%84%E4%B8%BB%E9%A0%81%E6%96%B0%E5%A2%9E%E3%80%8C%E6%9B%B4%E6%8F%9B%E6%99%82%E6%AE%B5%E3%80%8D%E5%8A%9F%E8%83%BD2)

File: src/components/appointment/translation.tsx [hunk 1]
```diff
@@ -9,6 +9,10 @@ const appointmentMessages = {
     finished: { id: 'appointment.*.finished', defaultMessage: '已結束' },
   }),
   AppointmentCard: defineMessages({
+    periodDurationAtMost: {
+      id: 'appointment.text.periodDurationAtMost',
```
Comment (Ponchimeow) [inline src/components/appointment/translation.tsx:13]: use the component name as the middle suffix.

'appointment.AppointmentCard.periodDurationAtMost'

File: src/pages/MemberPage/AppointmentPlanCollectionBlock.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const AppointmentPlanCollectionBlock: React.VFC<{ memberId: string }> = ({ membe
       <List>
         {enrolledAppointments.map(appointmentEnrollment => (
           <div key={appointmentEnrollment.orderProductId} className="mb-4">
-            <AppointmentCard {...appointmentEnrollment} onRefetch={refetchEnrolledAppointments} />
+            <AppointmentCard {...appointmentEnrollment} memberId={memberId} onRefetch={refetchEnrolledAppointments} />
```
Comment (Ponchimeow) [inline src/pages/MemberPage/AppointmentPlanCollectionBlock.tsx:24]: nitpick: I prefer using list props over spread operator.

**Review by Ponchimeow — CHANGES_REQUESTED** [review]
no big issue, just fix the translation

File: src/components/appointment/translation.tsx (hunk 1 above)
Comment (LoisChen68) [reply src/components/appointment/translation.tsx:13]: Eagle-eyed! Adjusted.

File: src/pages/MemberPage/AppointmentPlanCollectionBlock.tsx (hunk 2 above)
Comment (LoisChen68) [reply src/pages/MemberPage/AppointmentPlanCollectionBlock.tsx:24]: I agree, that using list props does indeed enhance readability. It has been adjusted.

**Pull request merged**